lib/litani_report.py

```diff
@@ -547,9 +547,10 @@ def render(run, report_dir):
 
     pipe_templ = env.get_template("pipeline.jinja.html")
     for pipe in run["pipelines"]:
-        PipelineDepgraphRenderer.render(
-            render_root=temporary_report_dir,
-            pipe_url=pathlib.Path(pipe["url"]), pipe=pipe)
+        if draw_dependency_graph:
```

Comment:
I suggest to get #55 merged. Once you address [this comment](https://github.com/awslabs/aws-build-accumulator/pull/55#discussion_r726550269), you might find you can use the same mechanism here